# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../udev/hotfix-notglibc-__s32-undefined.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Build error on mips32/dietlibc: __s32 undefined
klibc != !glibc, we are using dietlibc/uclibc as well!

--- udev-103/udev_sysdeps.h.orig	2006-10-20 14:43:35.000000000 +0200
+++ udev-103/udev_sysdeps.h	2006-12-11 10:32:16.000000000 +0100
@@ -104,7 +104,7 @@
 }
 #else
 /* needed until /usr/include/sys/inotify.h is working */
-#ifndef __GLIBC__
+#ifdef __KLIBC__
 #include <sys/inotify.h>
 #else
 static inline int inotify_init(void)
